posthog/test/base.py

```diff
@@ -231,21 +232,25 @@ def fn_with_materialized(self, *args, **kwargs):
                 materialize("events", prop)
             for prop in person_properties:
                 materialize("person", prop)
+                materialize("events", prop, table_column="person_properties")
 
             try:
                 with self.capture_select_queries() as sqls:
                     fn(self, *args, **kwargs)
             finally:
                 for prop in event_properties:
-                    column_name = get_materialized_columns("events")[prop]
+                    column_name = get_materialized_columns("events")[(prop, "properties")]
                     sync_execute(f"ALTER TABLE events DROP COLUMN {column_name}")
                 for prop in person_properties:
-                    column_name = get_materialized_columns("person")[prop]
+                    column_name = get_materialized_columns("person")[(prop, "properties")]
                     sync_execute(f"ALTER TABLE person DROP COLUMN {column_name}")
 
+                    column_name = get_materialized_columns("events")[(prop, "person_properties")]
+                    sync_execute(f"ALTER TABLE events DROP COLUMN {column_name}")
+
             if verify_no_jsonextract:
                 for sql in sqls:
-                    self.assertNotIn("JSONExtract(properties", sql)
+                    self.assertNotIn("JSONExtract", sql)
```

Comment:
We moved to JSONExtractRaw ages ago, so this was giving false negatives! A few tests will start failing because of this, and we should `verify_no_jsonextract=False` there if its a groups materialisation test (we never supported groups materialisation before it seems), or fix it if not :)